lib/wegift/models/remote_code.rb

```diff
@@ -11,16 +11,8 @@ class RemoteCode < Response
     attr_accessor :amount, :barcode_format, :barcode_string, :code, :expiry_date,
       :pin, :type
 
-    def self.get(url)
-      uri = URI(url)
-      conn = Faraday.new(url: "#{uri.scheme}://#{uri.host}")
-
-      response =  conn.get("#{uri.path}?format=json") do |req|
-        req.headers['Content-Type'] = 'application/json'
```

Comment:
Although not required (wegift returns `application/json` anyway when we pass `format=json`) I would still leave this for correctness and also because they might "fix" this behaviour without us knowing.

Reply:
We are not sending any content, I think it's not needed.

Reply:
While an Accept header would makes sense. But actually, I don't think they "fix" anything by accident :joy:

Reply:
Aye, This should be safe enough :+1: